fiaas_deploy_daemon/deployer/version_exporter.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+# Copyright 2017-2019 The FIAAS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from prometheus_client import Gauge
+
+
+class VersionExporter(object):
+    """ Exports the currently running FDD version """
+    version_gauge = Gauge("deployer_fdd_version", "Current FDD version",["app_name", "version"])
+
+    def fdd_version(self, app_spec):
+        self.version_gauge.labels(app_spec.name, app_spec.version).set(1)
```

Comment:
This will create a metric with the name and version of the deployed apps as labels, not the version of FDD that is running.